app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java

```diff
@@ -146,7 +146,7 @@ private String titleFor(User user) {
     }
 
     public class LovesPagerAdapter extends SmartFragmentStatePagerAdapter {
-        private String tabTitles[] = { "Received", "Sent" };
+        private String tabTitles[] = { "Sent", "Received" };
```

Comment:
Not a blocker for a merge, but would be nice to extract these into *strings.xml*